regolith/builders/publistbuilder.py

```diff
@@ -76,6 +76,26 @@ def latex(self):
                 cat_grants = cat_grants + "_" + g
             filestub = filestub + "{}".format(cat_grants)
             qualifiers = qualifiers + " from Grant{} {}".format(pl, text_grants)
+        if self.rc.kwargs:
+            kw = True
+            kwargs = self.rc.kwargs
+            if isinstance(kwargs, str):
+                key, value = kwargs.split(':', 1)  # Split only at the first colon
+                if key == 'facilities':
+                    facilities = value
+                    filestub = filestub + "_facility_{}".format(facilities)
```

Comment:
ok here, but please use f-strings for formatted strings in future

Reply:
will do!